### ISSUE TITLE
Mobile Browser Swipes rather than Horizontal Scroll

### ISSUE THREAD

**Issue by bakerboy448** [Radarr/Radarr#5731]
**Describe the bug**
Mobile horizontal scrolling acts as a swipe (left/right arrow key) and causes issues on mobile devices

**To Reproduce**
"I most use access via a iPad or iPhone (mostly ipad). And under the search section of a movie (so after u click on the search tab in movie) I scrolled down thru the selection but I then have to slide to the right to be able to see the qualities and to click download but instead because there is the new swipe feature it won’t allow me to and instead switches over to the next movie."

**Expected behavior**
Ignore swipes on small screens to facilitate horizontal touch scroll

**Screenshots**

**Platform Information (please complete the following information):**
 - OS: Agnostic
 - Docker: Agnostic
 - Mono or.NET Core Version: Agnostic
 - Browser and Version: Agnostic
 - Radarr Version Agnostic (3.0.2)
 - Radarr Branch Agnostic

**Trace Logs**
N/A

Reference:
https://discord.com/channels/264387956343570434/264388019585286144/798747349891416065

[AB#663](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/663)

**Comment by agneevX** [comment]
Yep, this is making horizontal scrolling in interactive search next to impossible on touch devices, especially iPads.

**Comment by agneevX** [comment]
Still an issue.

**Comment by Ramshackles** [comment]
Would love a fix for this, often use my ipad in the couch and its not working well

**Comment by Rouzax** [comment]
Also on Firefox on Android tablet.
Is there a setting somewhere to disable swipe to next movie?

**Comment by austinwbest** [comment]
There isn't. There is also a PR for a change to the details page completely so not sure how that factors in yet. We'll have to see

**Comment by Extremity** [comment]
Same issue on Chrome on an Android tablet, phone, etc. It's an issue regardless of browser/platform, as long as it's a touch device. The easiest solution seems to be a setting to disable swipe.

Does anybody actually use the horizontal swiping? I would if I had some control over it, ie options to have it go to the next item on the home page based on your current sort method rather than purely alphabetically. It seems kind of useless to me in its current form and is definitely more a hindrance than anything else, but I'm assuming somebody else out there must have a use for it. Right?

**Comment by austinwbest** [comment]
> Same issue on Chrome on an Android tablet, phone, etc. It's an issue regardless of browser/platform, as long as it's a touch device. The easiest solution seems to be a setting to disable swipe.
> 
> Does anybody actually use the horizontal swiping? I would if I had some control over it, ie options to have it go to the next item on the home page based on your current sort method rather than purely alphabetically. It seems kind of useless to me in its current form and is definitely more a hindrance than anything else, but I'm assuming somebody else out there must have a use for it. Right?

You're mixing things (feature request inside a bug report) but the option has a pending PR.. https://github.com/Radarr/Radarr/pull/7164

**Comment by majick** [comment]
To celebrate the two year and four month anniversary of this bug, I have posted [a $100 USD bounty](https://app.bountysource.com/issues/96287543-mobile-browser-swipes-rather-than-horizontal-scroll) against it.

**Comment by sashadev-sky** [comment]
@majick it looks like this bug is not present in the latest versions of Radarr. The swipe functionality is only on the image and not the search area. 

The bug noted in #6113 is still there, though, for text on the image.

**Comment by bakerboy448** [comment]
confirmed this appears to be fixed - like as a side effect of other recent UI changes or typescript conversions

 4.5.2.7300

**Comment by Extremity** [comment]
This is still occuring specifically for me; when browsing search results on a movie page, when trying to scroll left/right on the results I end up switching the movie I'm viewing to the next alphabetically.

Chrome 114.0.5735.196 (Android, Fold 3)
Radarr: 4.6.4.7568 (Running on Win 10)

I'm wondering if this is because I'm viewing Radarr from a Galaxy Fold 3; if this is specifically "fixed" for mobile, could Radarr not be considering it a mobile device? I do not have "view as desktop" enabled in Chrome. 

I've noticed that on the smaller, outer screen, my user agent data is:

Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Mobile Safari/537.36

But on the inner screen, it's:

Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

-- which is probably the issue. While a toggle would be really helpful it seems like it's not something you guys are willing to do. With the increased use of larger touchscreen devices that blur the line between "mobile" and "desktop" UI elements, is there anything that can be done to limit this behavior? It is effectively impossible for me to browse through search results otherwise.

**Comment by Extremity** [comment]
> confirmed this appears to be fixed - like as a side effect of other recent UI changes or typescript conversions
> 
> 4.5.2.7300
> 
> text selection to for the movie header can be tracked in #6113

This is still occuring for me, but it may be device-specific; when browsing search results on a movie page, when trying to scroll left/right on the results I end up switching the movie I'm viewing to the next alphabetically.

Chrome 114.0.5735.196 (Android, Fold 3)
Radarr: 4.6.4.7568 (Running on Win 10)

I'm wondering if this is because I'm viewing Radarr from a Galaxy Fold 3; if this is specifically "fixed" for mobile, could Radarr not be considering it a mobile device? I do not have "view as desktop" enabled in Chrome. 

I've noticed that on the smaller, outer screen, my user agent data is:

Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Mobile Safari/537.36

But on the inner screen, it's:

Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

-- which is probably the issue. While a toggle would be really helpful it seems like it's not something you guys are willing to do. With the increased use of larger touchscreen devices that blur the line between "mobile" and "desktop" UI elements, is there anything that can be done to limit this behavior? It is effectively impossible for me to browse through search results otherwise.

**Comment by bakerboy448** [comment]
What's the screen sizes / resolutions of both? It's likely hitting a weird spot

**Comment by Extremity** [comment]
According to https://www.webfx.com/tools/whats-my-browser-size/ - 

Outer screen: 374*925
Inner Screen: 786*847

Keep in mind that this can vary based on scaling settings set in the main Android settings. I ran into a similar issue with a site I'm building for a client, the number of @media width CSS entries is absolutely ridiculous.

There might be a little cheat you could consider by checking the Platform/SEC-CH-UA-PLATFORM HTTP header to see if it equals "Android" (and, if you want to be more specific, you could then check the model/SEC-CH-UA-MODEL header and substring it to see if the first three characters are "SU-"). This way you could have behavior for all Android devices which are going to be phones/tablets 99% of the time, or if for some reason you want to narrow it down more the model check should better limit it to handheld devices.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-UA-Platform

**Comment by planetrocky** [comment]
Still an issue.  Can’t copy path from details on mobile, as any finger movement to highlight text swipes forwards/backwards through movies.

Agee with previous poster. I don’t see the need for any horizontal swiping.

This swiping is probably linked to the other bug where it jumps to next movie when editing details.

**Comment by Extremity** [comment]
I believe device type is relevant here - I had this problem in the past (I previously posted in this issue) and while the horizontal scrolling still exists for me on mobile and I do believe it's completely useless as a whole - if it adhered to your current sorting method it would feel less pointless - I can at least select text now. 

I'm use Android devices, so selecting text may be different than Apple devices. When you push and hold on a word to select it, you're then given two small "bubbles" you can drag to expand the selection in either direction; because of this, those touch inputs are not actually passed to the DOM which is "beneath" them. I thought this may be why it works for some and not others, but if I select text by pushing down and holding and then dragging to the side (never releasing, therefore never getting or using the "bubbles") it still works and does not swipe. If I make a left or right motion in any other circumstances it does swipe and changes to another movie.

It would probably be helpful to state exactly which mobile device you're on when posting here. For instance, for the above post I don't know what device they're using and therefore cannot test and see if I can replicate the behavior. 

As for the reply that said this was previously fixed, that fix says it disables swiping "when a modal is open." Unless I'm misunderstanding what that means I don't believe it helps when you're just at the base details page?
